Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
```

Comment:
Could we add validation to GitHub Actions for this? E.g. for CocoaPods build, `pod spec lint` will build the pod and check the podspec is valid.

Reply:
Added validate-swiftpm.yml